### DISCUSSION TITLE
Ebird

### DISCUSSION THREAD

**Discussion by matthew73210** [mcguirepr89/BirdNET-Pi#812] [Ideas]
Hey,

Just a quick idea, any chance you could add a link to the ebird site of birds that are pinged? I usually hop over there to double check birds I'm not used to seeing. Green woodpecker masquerading as a bold...

Cheers

**Comment by ehpersonal38** [comment]
Sorry can you clarify this a bit more?

**Reply by matthew73210** [reply]
Hello,

At the moment, on a birds recording you have added a link the the wiki page. I was wondering if you could do the same with ebird. As they have recordings of birds, it's an easy way to cross reference.

**Reply by gerdesj** [reply]
I've just taken a quick look at that site and book marked it.  Cornell Uni is well respected and there must be loads of other academic sites, so why stop at ebird.
